src/composables/queries/usePoolQuery.ts

```diff
@@ -157,8 +159,20 @@ export default function usePoolQuery(
       currency.value
     );
 
-    console.log('pool', { onchain: onchainData, ...decoratedPool });
-    return { onchain: onchainData, ...decoratedPool };
+    let unwrappedTokens: Pool['unwrappedTokens'];
+
+    if (isStablePhantomPool && onchainData.linearPools != null) {
+      unwrappedTokens = Object.entries(onchainData.linearPools).map(
+        ([, linearPool]) => linearPool.unwrappedTokenAddress
+      );
+    }
```

Comment:
We're already modifying the pool object with two new arrays of pool tokens in `getLinearPoolAttrs` could you not just move the additional construction of unwrappedTokens here too: https://github.com/balancer-labs/frontend-v2/blob/0f3c9c1569df580b109e4d1563ffbdc7faa4ea5b/src/composables/queries/usePoolQuery.ts#L95-L100

Reply:
Right, but `onchainData ` (where the `unwrappedTokenAddress` is) is computed after the `getLinearPoolAttrs` method. (so it doesn't exist there yet).

So you mean we want to compute `onchainData` before calling `getLinearPoolAttrs` right?

Reply:
Ahh crap, sorry! You're right. Would be good if we could get these tokens from the subgraph call to the linear pools in `getLinearPoolAttrs` but I guess we can't. We'll have to clean all this up later.

Reply:
Yeah I agree.

Are you happy for this to be merged for now?